code/modules/codex/categories/category_substances.dm

```diff
@@ -61,14 +61,15 @@
 				var/decl/material/R = chemical
 				chems += "[initial(R.name)] ([mat.dissolves_into[chemical]*100]%)"
 			var/solvent_needed
-			if(mat.dissolves_in <= MAT_SOLVENT_NONE)
-				solvent_needed = "any liquid"
-			else if(mat.dissolves_in <= MAT_SOLVENT_MILD)
-				solvent_needed = "a mild solvent, like water"
-			else if(mat.dissolves_in <= MAT_SOLVENT_MODERATE)
-				solvent_needed = "a moderately strong solvent, like acetone"
-			else if(mat.dissolves_in <= MAT_SOLVENT_STRONG)
-				solvent_needed = "a strong solvent, like sulphuric acid"
+			if(mat.dissolves_in != MAT_SOLVENT_IMMUNE)
+				if(mat.dissolves_in <= MAT_SOLVENT_NONE)
+					solvent_needed = "any liquid"
+				else if(mat.dissolves_in <= MAT_SOLVENT_MILD)
+					solvent_needed = "a mild solvent, like water"
+				else if(mat.dissolves_in <= MAT_SOLVENT_MODERATE)
+					solvent_needed = "a moderately strong solvent, like acetone"
+				else if(mat.dissolves_in <= MAT_SOLVENT_STRONG)
+					solvent_needed = "a strong solvent, like sulphuric acid"
 			material_info += "<li>It can be dissolved with [solvent_needed] solvent, producing [english_list(chems)].</li>"
```

Comment:
Shouldn't this be gated under solvent immunity check to?
Would result in 'it can be dissolved with  solvent' otherwise